modules/two_factor_authentication/lib/open_project/two_factor_authentication/token_strategy_manager.rb

```diff
@@ -114,7 +114,7 @@ def enforced_by_configuration?
       def merge_with_settings!(config)
         config['active_strategies'] ||= []
         # Always enable totp if nothing is enabled
-        config['active_strategies'] << :totp if add_default_strategy?(config)
+        config['active_strategies'] += %i[totp] if add_default_strategy?(config)
```

Comment:
```suggestion
        config['active_strategies'] += %i[totp webauthn] if add_default_strategy?(config)
```

Webauthn should be a default strategy so people can use it directly

Reply:
Did not want to put any new defaults on people, but makes sense. reverted bc5665a57d20b41f00f3861ad1fb5c2d71cfa491